docs/Third-party.md

```diff
@@ -1,281 +1,268 @@
+        - `source FILE` : used to read/execute a FILE, for example one where configurations are set
 
-Clinica requires dcm2niix version `1.0.20190902` or later.
+2. In order to define these variables permanently, you need to manually edit the configuration file associated to your shell.
 
-## Pipeline-specific interfaces
+    If you are using `bash`, it can be `~/.bashrc` or `~/.bash_profile`, if you are using `zsh`, it should be `~/.zshrc`.
 
-Not all the following dependencies are necessary to install and run Clinica.
-You may want to only install the software packages used by certain pipelines of Clinica.
-Pipelines' specific dependencies are listed below:
-
-|                          | ANTs | Convert3D | FreeSurfer | FSL | ITK | Matlab | MRtrix3 | PETPVC | SPM |
-|:-------------------------|:----:|:---------:|:----------:|:---:|:---:|:------:|:-------:|:------:|:---:|
-| `t1-volume-*`            |      |           |            |     |     |    x   |         |        |  x  |
-| `t1-freesurfer`          |      |           |     x      |     |     |        |         |        |     |
-| `dwi-preprocessing-*`    |   x  |     x     |            |  x  |     |        |    x    |        |     |
-| `dwi-dti`                |   x  |           |            |  x  |     |        |    x    |        |     |
-| `dwi-connectome`         |   x  |           |     x      |  x  |     |        |    x    |        |     |
-| `pet-surface`            |      |           |            |  x  |  x* |        |         |   x*   |  x  |
-| `pet-volume`             |      |           |            |     |  x* |    x   |         |   x*   |  x  |
-| `statistics-surface`     |      |           |            |     |     |    x   |         |        |     |
-| `machine-learning-*`     |      |           |            |     |     |        |         |        |     |
-
-!!! note "CAT12 toolbox"
-    Starting from Clinica `v0.3.7`, the [**CAT12**](http://dbm.neuro.uni-jena.de/cat/) toolbox is no longer needed for the `t1-volume` and `pet-volume` pipelines.
-    For previous versions of Clinica, you will need to download the latest version of the toolbox [here](http://dbm.neuro.uni-jena.de/cat/index.html#DOWNLOAD) and follow the instructions to ensure that your `cat12` folder is located in your `spm/toolbox` folder.
+!!! danger "Provided command lines should be adapted"
+    **Please** do not copy/paste the provided commands without adapting them to your system and without testing them.
+    Most paths provided here require to be adapted to your system and depend on how you installed the software.
+    
+    A good approach is to verify that the different paths you want to assign to a variable exist first : 
 
-_*You only need to install ITK if you plan to perform partial volume correction using PETPVC._
+    :material-arrow-right: If they do, then try running the `export` and `source` **(1.)** commands in your terminal. Verify that the software or the pipeline you want runs as expected.
```

Comment:
The `:material-arrow-right:` and `:material-arrow-right:` do not seem to render on the served html version.

Reply:
Thanks for spotting it !